lib/ansible/utils/display.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, verbosity=0):
                 cmd = subprocess.Popen([self.b_cowsay, "-l"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
                 (out, err) = cmd.communicate()
                 self.cows_available = set([to_text(c) for c in out.split()])
-                if C.ANSIBLE_COW_WHITELIST:
+                if any(C.ANSIBLE_COW_WHITELIST):
```

Comment:
shouldn't we keep the previous check also (make it an `and`) to avoid `None` values instead of list or list of empty items?

Reply:
Probably

Reply:
Done.